task/gradle/0.3/gradle.yaml

```
@@ -0,0 +1,70 @@
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name: gradle
+  labels:
+    app.kubernetes.io/version: "0.3"
+  annotations:
+    tekton.dev/pipelines.minVersion: "0.17.0"
+    tekton.dev/displayName: Gradle
+    tekton.dev/categories: Build Tools
+    tekton.dev/tags: build-tool
+    tekton.dev/platforms: "linux/amd64,linux/s390x,linux/ppc64le"
+spec:
+  description: >-
+    This Task can be used to run a Gradle build.
+
+  workspaces:
+    - name: source
+      description: The workspace consisting of the gradle project.
+    - name: gradle-config
+      description: The workspace consisting of the custom gradle properties provided by the user.
+      optional: true
```

instead of doing via script, can we have a mountPath here?

```suggestion
    - name: gradle-config
      description: The workspace consisting of the custom gradle properties provided by the user.
      optional: true
      mountPath: /home/gradle
```

---

for reference https://tekton.dev/docs/pipelines/workspaces/#using-workspaces-in-tasks

---

In the example code I have used a configMap to store the gradle.properties file. In our own pipelines we use a secret as source.
Gradle needs a /home/gradle that is writable so I don't think a mountPath will work.